spec/services/solidus_bolt/users/sync_payment_sources_service_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe SolidusBolt::Users::SyncPaymentSourcesService, :vcr, :bolt_configuration do
+  subject(:sync_payment_sources) do
+    described_class.call(user: user, access_token: ENV.fetch('BOLT_ACCESS_TOKEN', 'Fake_access_token'))
+  end
+
+  let(:user) { create(:user) }
+
+  describe '#call' do
+    it 'creates a new payment source with card ID' do
+      sync_payment_sources
+      bolt_payment_source = SolidusBolt::PaymentSource.last
+      expect(bolt_payment_source.card_id).to eq('CAgP9uNSEZBNL')
```

Comment:
You can't use static values here, otherwise, it requires to be changed anytime the cassettes will be recorded.

Reply:
fixed in 29164185557f51b274e142a0ff3f2ca9de035862